src/test/java/ir/sahab/kafka/reader/KafkaProtoParquetWriterTest.java

```diff
@@ -217,6 +224,28 @@ public void testDirectoryDateTimePattern() throws Throwable {
         assertThat(actual, containsInAnyOrder(messages.toArray(new SampleMessage[0])));
     }
 
+    /**
+     * This function commits uncommitted records at the end of the tests.
+     * It should be run at the end of the tests to make sure no record of current test appears in the next test.
```

Comment:
```suggestion
     * Consumes the remaining records of the topic and commits the last offset of partitions.
```
In Javadoc of the method describe 'what' it does. It is up to the caller to decide when to call it.

Reply:
Done